gcore/src/msg.rs

```diff
@@ -21,7 +21,6 @@ use crate::{MessageId, ProgramId};
 
 mod sys {
```

Comment:
Following the idea of this pull request, wouldn't it be reasonable also to move `gr_read()` and `gr_size()` to `gcore/exec` mod?

Reply:
In my opinion, both `gr_read` and `gr_size` are related to the message, so there is no reason to move them.